kubernetes/pybot/overlays/staging/deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pybot
+spec:
+  template:
+    spec:
+      containers:
+        - name: app
+          image: operationcode/pybot:staging
+          env:
+            - name: BACKEND_URL
+              value: https://api.staging.operationcode.org
+            - name: BACKEND_USERNAME
+              value: Pybot@test.test
+            - name: COMMUNITY_CHANNEL
+              value: GJBH2A60N
+            - name: ENVIRONMENT
+              value: staging
+            - name: MENTOR_CHANNEL
+              value: GHXU0NAAF
+            - name: MODERATOR_CHANNEL
+              value: GHXU0FRK5
+            - name: PYBACK_HOST
+              value: pybot.operationcode.org
```

Comment:
If I understand what this does correctly, I _think_ this would be the new URL of pybot now - which we create via the ingress rules - `pybot-staging.k8s.operationcode.org`